### ISSUE TITLE
hasAtrribute never finding an infraction

### ISSUE THREAD

**Issue by ryandcarter** [mule-lint/mule-lint#19]
Hi, I have downloaded the latest 0.6.0 snapshot and it runs fine. dash cased, camel cased all works.

But hasAttribute never seems to find infractions.

element 'logger' hasAttribute 'blabla'

**Comment by ryandcarter** [comment]
Actually, it all depends on what the first line in the rules.txt is:
It only seems to run the first line in the rules.txt file. This one finds  logger infractions but not flows:
version '0.0.1'

element 'logger' hasAttribute 'category'
flows are dash cased

This finds flow infractions but not loggers:
version '0.0.1'

flows are dash cased
element 'logger' hasAttribute 'category'

**Comment by cgorshing** [comment]
Thanks for the report! I'll work on this and get back to you soon - thanks again!

**Comment by cgorshing** [comment]
Looks like the last rule isn't added correctly to the list (fail).

So a quick workaround would be:
```
version '0.0.1'

flows are dash cased
element 'logger' hasAttribute 'category'
element 'logger' hasAttribute 'category'
```

Working on the fix.

**Comment by cgorshing** [comment]
I've got these changes in and will try and cut a release tonight. I might not as my key previously used has expired and I created a new one, so not sure what that process looks like for maven central with a new key.

**Comment by cgorshing** [comment]
Got it released - can you try 0.5.1 please?

https://repo1.maven.org/maven2/org/nuisto/mule-lint/0.5.1/
